include/basic_types.h

```diff
@@ -8,8 +8,22 @@
 
 #include <stdint.h>
 #include <arch/types.h>
+#include <assert.h>
 
-/* arch/types.h is supposed to define word_t and _seL4_word_fmt */
+/* arch/types.h defines wordRadix, wordBits is 2^wordRadix. */
+#ifdef CONFIG_ARCH_RISCV
+/* Unfortunately, the proofs for RISC-V are build around using BIT() and need to
+ * be updated to unify the definitions.
+ */
+#define wordBits    BIT(wordRadix)
+#else
+#define wordBits    (1 << wordRadix)
```

Comment:
I don't think this is clearer.  wordBits is an arch or mode specific definition, so it's not unclear to find it in the arch/types.h or arch/mode/types.h header next to wordRadix which is a related closely related definition.

Reply:
My intention here is unifying definition as much as possible - or prepare this at least and ad comment what is still blocking this. Also, I try to provide some guidance, which definition is preferred (seems `(1 << wordRadix)`) in case there is another architecute/mode arising.
I agree that `wordBits` is an arch/mode type, but is seems we build this upon `wordRadix`, which is the only thing an architecture has to provide. Assuming that `wordBits` is of the from `2^n` might be a bit of gambling since there have been other architectures in the past with 7 or 9 or 26 bit words. But I really wonder if they ever come back or sel4 is ever supposed to run there.